### DISCUSSION TITLE
友情链接 - Alan|前端博客

### DISCUSSION THREAD

**Discussion by giscus** [3Alan/site#15] [Announcements]
# 友情链接 - Alan|前端博客

Alan 的友情链接

https://www.alanwang.site/friends

**Comment by 3Alan** [comment]
回复示例
```js
{
  title: 'Alan',
  avatar: 'https://avatars.githubusercontent.com/u/49978973?v=4',
  url: 'https://www.alanwang.site/',
  description: '此刻想举重若轻，之前必要负重前行',
  tags: ['前端开发']
}
```

**Comment by kuizuo** [comment]
{
  title: 'Kuizuo',
  avatar: 'https://kuizuo.cn/img/logo.png',
  url: 'https://kuizuo.cn',
  description: '记录所学知识，领略编程之美',
  tags: ['前端开发', 'ts全栈']
}

**Reply by 3Alan** [reply]
已添加，博客写的挺好👍

**Comment by fantingsheng** [comment]
{
  title: '太空编程',
  avatar: 'https://spacexcode.com/img/share.png',
  url: 'https://spacexcode.com',
  description: '分享硬核的前端编程知识',
  tags: ['前端开发']
}

**Reply by 3Alan** [reply]
已添加，期待添加本站为友链

**Reply by fantingsheng** [reply]
网站底部已添加

**Comment by Casta-mere** [comment]
谢大佬

```jsx
{
  title: "Castamere",
  avatar: "https://github.com/Casta-mere.png",
  url: "https://www.castamerego.com/",
  description: "代码美学",
  tags: ["全栈开发", "python开发"],
}
```

**Reply by 3Alan** [reply]
已添加

**Comment by ans-gab** [comment]
{
  title: '法考笔记',
  avatar: 'https://avatars.githubusercontent.com/u/73372448',
  url: 'https://law.vuestyle.com',
  description: '法律知识网站',
  tags: ['法律知识']
}
